onnx/checker.cc

```diff
@@ -223,20 +223,23 @@ void check_attribute(
 #undef check_singular_field
 #undef check_repeated_field
 
-  if (ctx.is_main_graph()) {
-    if (used_fields != 1) {
-      fail_check(
-          "Attribute (name: ",
-          attr.name(),
-          ") should contain one and only one value field.");
-    }
-  } else {
+  // Normally, used_fields is expected to be 1.
+  // In proto3, when the value to be set is type default value (say 0 for int), used_fields may be 0.
+  if (used_fields > 1) {
+	  fail_check(
+		  "Attribute (name: ",
+		  attr.name(),
+		  ") should contain less than one value field.");
```

Comment:
"should not contain more than one value field"

Reply:
or "contains more than one value field."

Reply:
Fixed.